src/loaders/ObjectLoader.js

```diff
@@ -550,13 +583,29 @@ Object.assign( ObjectLoader.prototype, {
 					break;
 
 				case 'Mesh':
+				case 'SkinnedMesh':
 
 					var geometry = getGeometry( data.geometry );
 					var material = getMaterial( data.material );
+					var skeleton = getSkeleton( data.skeleton );
+
+					// override bones if skeleton exists
+					if ( skeleton !== undefined ) {
+
+						geometry.bones = skeleton.bones;
+						geometry.boneInverses = skeleton.boneInverses;
+
+					}
 
 					if ( geometry.bones && geometry.bones.length > 0 ) {
 
-						object = new SkinnedMesh( geometry, material );
+						var useVertexTexture = ( skeleton !== undefined ) ? skeleton.useVertexTexture : undefined;
+
+						object = new SkinnedMesh( geometry, material, useVertexTexture );
+
+						if ( data.bindMode !== undefined ) object.bindMode = data.bindMode;
+						if ( data.bindMatrix !== undefined ) object.bindMatrix.fromArray( data.bindMatrix );
+						object.updateMatrixWorld( true );
```

Comment:
@mrdoob 
Currently in my PR, here instantiates `Mesh` if `type` is "SkinnedMesh" without `.bone`
But should `SkinnedMesh` be instantiated as `type` specifies?
What do you think?